src/ethereum_spec_tools/evm_tools/fixture_loader.py

```diff
@@ -23,6 +23,14 @@
 from ethereum_spec_tools.forks import Hardfork
 
 
+class UnsupportedTx(Exception):
+    """Exception for unsupported transactions"""
+
+    def __init__(self, encoded_params: bytes, error_message: str) -> None:
+        self.encoded_params = encoded_params
```

Comment:
Something like this might render better in a backtrace:

```suggestion
        super.__init__(error_message)
        self.encoded_params = encoded_params
```